src/Juvix/Backends/LLVM/Codegen/Graph.hs

```diff
@@ -259,7 +295,31 @@ defineRewire nodePtrType = Block.defineFunction Type.void "rewire" args $
         (numPorts, "port_two")
       ]
 
-delNode = undefined
+deAllocateNode ∷ Define m ⇒ Operand.Operand → Type.Type → m Operand.Operand
+deAllocateNode nodePtr nodePtrType = do
+  node ← load (nodeType nodePtrType) nodePtr
```

Comment:
So, reading this I am sorta wondering why these aren't contiguous

If they were contiguous (even if they had separate pointers) we could free them with one `free`, one would suppose.

We can worry about this later though

Reply:
I'd say let's make a TODO of this, I actually thought of this myself after writing it.

However, I decided to can it for the time being, as I'm not the most confident in the free and malloc behaviors when interacting with LLVM code.

Reply:
Added a TODO